packages/@okta/vuepress-site/docs/reference/password-hook/index.md

```diff
@@ -161,5 +161,5 @@ Normally, if your external service responds to Okta indicating that the credenti
 
 Because of the posssiblity of read-only mode, permanent deletion of user passwords from your existing user store should not be performed until success of the password migration can be verified. An Okta System Log [Event](/docs/reference/api/event-types/), `user.import.password`, is available for this purpose. An event of this type is created every time a Password Import Inline Hook is fired, with its `Event.Outcome` property providng a status of `FAILURE` or `SUCCESS` for the password import operation. If the status is `SUCCESS`, Okta has successfully saved the end user's password, and it's safe to delete it from your previous user store.
```

Comment:
Is this technically accurate @kenwang-okta @rameshbhojan-okta ? I was under the assumption that password import hooks work in Read-Only mode?
I agree we should still have a note around not deleting password until event is fired.

Reply:
Yes, technically we import it even in RO mode, and only don't do anything in the `SAFE` mode. So, maybe we shouldn't specify `read-only mode` there and instead just state it like generically something along the lines of `Depending on the mode of operation`

Reply:
@rameshbhojan-okta, changed to:
"Because of the possibility of your org being in a special service mode, permanent deletion of user passwords from your existing user store should not be performed until success of the password migration can be verified."